website/docs/developer/research/research-localstack.md

```diff
@@ -371,10 +402,9 @@ Here is the output after running `terraform apply`
     aws_lambda_function.test_lambda: Creation complete after 6s [id=localstack-lambda]
 
     Apply complete! Resources: 8 added, 0 changed, 0 destroyed.
-
-
+```
 ### IDE for Localstack
-- Currently there is an IDE named [Commandeer](https://getcommandeer.com/) that can be used to maintain localstack environments but requires to buy a premium version to use. This could be something to look into in the future if we plan to buy a premium version.
+- Currently, there is an IDE named [Commandeer](https://getcommandeer.com/) that can be used to maintain localstack environments but requires to buy a premium version to use. This could be something to look into in the future if we plan to buy a premium version.
```

Comment:
change `requires to buy a premium version to use` to `requires a licensed premium version of the software to use`